### PR TITLE
Update phoenix assets to 0.21.0

### PR DESCRIPTION
**Pull request by kulmann**

### CODE REVIEW

**Review by LukasHirt — APPROVED** [review]
LGTM :+1:

File: .drone.star [hunk 1]
```diff
@@ -521,110 +521,48 @@ def accountsUITests(ctx, phoenixBranch, phoenixCommitId, storage):
...
-          {
-            'name': 'gopath',
-            'path': '/srv/app'
-          },
-        ]
-      },
+    'steps':
+      generate('ocis') +
+      build() +
+      ocisServer(storage) + [
```
Comment (micbar) [inline .drone.star:527]: Can we really start the full server stack?

File: .drone.star [hunk 2]
```diff
@@ -521,110 +521,48 @@ def accountsUITests(ctx, phoenixBranch, phoenixCommitId, storage):
...
-          'STORAGE_LDAP_IDP': 'https://ocis-server:9200',
-          'PROXY_OIDC_ISSUER': 'https://ocis-server:9200',
-          'STORAGE_HOME_EXPOSE_DATA_SERVER': 'true',
-          'STORAGE_HOME_DATA_SERVER_URL': 'http://ocis-server:9155/data',
-          'STORAGE_DATAGATEWAY_PUBLIC_URL': 'https://ocis-server:9200/data',
-          'STORAGE_FRONTEND_PUBLIC_URL': 'https://ocis-server:9200',
-          'PHOENIX_WEB_CONFIG': '/drone/src/accounts/ui/tests/config/drone/ocis-config.json',
-          'KONNECTD_IDENTIFIER_REGISTRATION_CONF': '/drone/src/accounts/ui/tests/config/drone/identifier-registration.yml',
-          'KONNECTD_ISS': 'https://ocis-server:9200',
-          'ACCOUNTS_STORAGE_DISK_PATH': '/srv/app/tmp/ocis-accounts', # Temporary workaround, don't use metadata storage
```
Comment (micbar) [inline .drone.star:564]: This was needed before. maybe not more on that branch

Comment (kulmann) [reply .drone.star:564]: hm... maybe that's what I was trying to find. It **is** needed on this branch.

File: .drone.star (hunk 1 above)
Comment (kulmann) [reply .drone.star:527]: why not? I added the accounts UI to the web config that is being used for the ocisServer. With that it should be fit for use.

Comment (micbar) [reply .drone.star:527]: streamlines the setup, fine

**Review by micbar — APPROVED** [review]

**Review by refs — APPROVED** [review]

**Pull request merged**